### PR TITLE
rgw: install rgw scripts with common files rather than radosgw files

### PR DESCRIPTION
**Pull request by ivancich**
Update ceph.spec.in and debian install files so rgw-restore-bucket-index, rgw-orphan-list, rgw-gap-list, rgw-gap-list-comparator are installed with common files.

Fixes: https://tracker.ceph.com/issues/59078

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: ceph.spec.in [hunk 1]
```diff
@@ -2128,11 +2132,7 @@ fi
 %{_bindir}/radosgw-token
 %{_bindir}/radosgw-es
 %{_bindir}/radosgw-object-expirer
-%{_bindir}/rgw-gap-list
-%{_bindir}/rgw-gap-list-comparator
-%{_bindir}/rgw-orphan-list
```
[the remark this answers is not in the document]
Comment (ivancich) [reply ceph.spec.in:2133]: Does the common section mean that the scripts will be installed on each machine, no matter its designated purpose?

**Comment by cbodley** [comment]
@ivancich feel free to run just the rgw/upgrade subsuite. just try to make sure we get coverage of centos and ubuntu

**Comment by cbodley** [comment]
@ivancich the fix in https://github.com/ceph/ceph/pull/50644 only touches the qa directory, so you could reschedule your existing builds against that:
```
-s rgw/upgrade --suite-repo https://github.com/cbodley/ceph.git --suite-branch wip-59135
```

**Comment by ivancich** [comment]
QA run of rgw/upgrade suite:
* https://pulpito.ceph.com/ivancich-2023-03-23_15:44:21-rgw:upgrade-wip-eric-testing-1-distro-default-smithi/

File: ceph.spec.in (hunk 1 above)
[the remark this answers is not in the document]
Comment (ktdreyer) [reply ceph.spec.in:2133]: In the past I've seen issues when we move files on Debian packages. I think that's solved now that the Debian sub-packages strictly require the same versions. (Like RPMs do).

In other words, I think it's fine to move files like you've done here.

[the remark this answers is not in the document]
Comment (cbodley) [reply ceph.spec.in:2133]: wonderful, thanks @ktdreyer

**Pull request merged**